aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -505,6 +506,28 @@ Tensor sparse_to_dense(
   return dst.add_(self);
 }
 
+SparseTensor& copy_sparse_wrapper_(
+    Tensor& self,
+    const Tensor& src,
+    bool non_blocking) {
+  // TODO: Once copy_ is fully migrated to use dispatcher, handle named
+  // inference using dispatcher instead of doing it everywhere
+  auto maybe_outnames = namedinference::compute_broadcast_outnames(self, src);
```

Comment:
Is this making copy sparse more correct than it was before? Because in the old code it seems like names were just not propagated for sparse at all. I think I'd probably prefer that, mostly because it's not worth gooping up the code here.

Reply:
And then maybe you can use `copy_sparse_to_sparse_`; so instead of double dispatch (go to wrapper, than go to copy sparse to sparse), just inline the dispatch table of  `copy_sparse_to_sparse` into `copy`.

Reply:
Uh....I thought the sparse flow was going through name propagation. It went through `copy_` which did name prop and called `copy_impl` containing all implementations.
Please correct me if I'm wrong

Reply:
Blast, you're right

Reply:
Great. This PR would work then?